app/components/ch/components/DownloadList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <li>
+    <button
+      class="download-item"
+      :href="url"
+      target="_blank"
+      :download="title"
+      :id="`download-item-${id}`"
+    >
+      <SvgIcon icon="Download" size="xl" class="download-item-icon" />
+      <div>
+        <p class="download-item-title">
+          {{ title }}
+        </p>
+        <p v-if="description" class="download-item-description">
+          {{ description }}
+        </p>
+        <p class="download-meta-info">
+          <span class="download-item-size">
+            {{ type }}
+          </span>
+          <span class="download-item-meta-divider">|</span>
```

Comment:
As this is a visual separator only, isn't it better to drop the markup and do it with CSS?

Reply:
yep. done